ibm_catalog.json

```
@@ -0,0 +1,546 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-vsi-vpc",
+      "label": "[Add-ons Beta] Cloud automation for Virtual server instance on VPC",
```

To align with official docs:
```suggestion
      "label": "[Add-ons Beta] Cloud automation for virtual server instances for VPC",
```